biolink-model.yaml

```diff
@@ -6512,6 +6516,7 @@ slots:
     multivalued: true
 
   supporting data source:
+    aliases: ['supporting dataset']
```

Comment:
See my comments here: https://github.com/biolink/biolink-model/issues/1074#issuecomment-1228937425

Reply:
removed alias and added a second slot for 'study data set'